robot_smach_states/src/robot_smach_states/world_model.py

```diff
@@ -203,12 +204,16 @@ def __init__(self, robot, entityDes, objectIDsDes=None, searchArea="on_top_of",
                                                                                  entityDes),
                                        transitions={'unreachable': 'failed',
                                                     'goal_not_defined': 'failed',
-                                                    'arrived': 'SEGMENT'})
+                                                    'arrived': 'RISE'})
             else:
                 smach.StateMachine.add('NAVIGATE_TO_INSPECT', NavigateToObserve(robot, entityDes, radius=1.0),
                                        transitions={'unreachable': 'failed',
                                                     'goal_not_defined': 'failed',
-                                                    'arrived': 'SEGMENT'})
+                                                    'arrived': 'RISE'})
+
+            smach.StateMachine.add('RISE', RiseForInspect(robot, entityDes, searchArea),
```

Comment:
We might consider segmenting even if rising failed. Segmentation might still work, won't it?